flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinition.java

```diff
@@ -195,16 +195,31 @@ public Builder notDeterministic() {
             return this;
         }
 
+        /**
+         * Specifies that this {@link BuiltInFunctionDefinition} is implemented during code
+         * generation.
+         */
         public Builder runtimeProvided() {
             this.isRuntimeProvided = true;
             return this;
         }
 
+        /** Specifies the runtime class implementing this {@link BuiltInFunctionDefinition}. */
         public Builder runtimeClass(String runtimeClass) {
             this.runtimeClass = runtimeClass;
             return this;
         }
 
+        /**
+         * Specifies that this {@link BuiltInFunctionDefinition} will be mapped to a Calcite
+         * function.
+         */
+        public Builder runtimeDeferred() {
```

Comment:
nit: `runtimeTransformed` or `runtimeConverted`?

Reply:
I kind of like `runtimeDeferred` in the sense that the runtime implementation defers (to Calcite). You marked it a nit, so I'll be cheeky and ignore it for now, but feel free to make it a stronger opinion. :)